lib/StaticNarrative/exporter/app_processor.py

```diff
@@ -83,19 +92,23 @@ def _make_upa_dict(self, value, param_spec: dict):
                     for v in value:
                         if self._is_upa(v):
                             upas.append(v)
-                else:
-                    if self._is_upa(value):
-                        upas.append(value)
+                elif self._is_upa(value):
+                    upas.append(value)
```

Comment:
no actual code change but these lines are obviously not covered by the tests! D: